pallets/delegation/src/deprecated.rs

```diff
@@ -101,3 +101,60 @@ pub(crate) mod v1 {
 		}
 	}
 }
+
+pub(crate) mod v2 {
```

Comment:
I would propose to just get rid of the v1 migration instead. Else, I agree.

Reply:
I would personally not remove any migrations until we have added the KILT features to spiritnet. After that, I would be okay with removing them.

Reply:
The migrations should not get executed on spiritnet. Why keep them?

Reply:
Why though? When we add them to the Spiritnet, they come fresh and are independent of migrations anyway. I'd rather start fresh.

Reply:
We should always have the most recent storage version on all of our networks. So we only need to support migrations from `n-1` to `n` if `n` is the most recent version. Once we add `n+1` we remove `n-1`.

I would not reset the StorageVersion but keep the current versioning.

Reply:
Yes, that is my point. We can of course remove the old migrations as also Parity does, but we need to establish some process about how we are going to upgrade. So I was indirectly referring to this, since we do not have yet a defined way of making upgrades.

Reply:
The process is to not skip runtime upgrades. The migrations are in the runtime